packages/title-case/src/index.ts

```diff
@@ -76,6 +77,7 @@ export function titleCase(
     titleTerminators = TITLE_TERMINATORS,
     smallWords = SMALL_WORDS,
     wordSeparators = WORD_SEPARATORS,
+    lowerCaseSmallWords = false,
```

Comment:
Should this lower case at the end or beginning of a sentence?

Reply:
Hi Blake, so this is for the case where the input contains small words that are not in lowercase but the current implementation does not lowercase them. Regard your second question, I think it should not lowercase the beginning and ending of a sentence.